packages/legend-extension-dsl-service/src/components/studio/UpdateServiceQuerySetup.tsx

```diff
@@ -298,6 +312,11 @@ export const UpdateServiceQuerySetup = withUpdateServiceQuerySetupStore(
               setupStore.currentSnapshotService.path,
             ),
           ).catch(applicationStore.alertUnhandledError);
+          if (setupStore.isCurrentProjectNotConfigured) {
```

Comment:
With https://github.com/finos/legend-studio/pull/1506 merged, we should also account for that, the fact tthat